exporter/jaegerexporter/factory.go

```diff
@@ -19,27 +19,24 @@ import (
 	"fmt"
 
 	"go.opentelemetry.io/collector/component"
-	"go.opentelemetry.io/collector/config/configerror"
 	"go.opentelemetry.io/collector/config/configgrpc"
 	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/exporter/exporterhelper"
 )
 
 const (
 	// The value of "type" key in configuration.
 	typeStr = "jaeger"
 )
 
-// Factory is the factory for Jaeger gRPC exporter.
-type Factory struct {
```

Comment:
"// NewFactory creates a factory for Jaeger gRPC exporter." (similar for elsewhere)

Reply:
Done.